api/lib/handlers/projects/index.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var config = require('../../config.js').config;
+var pg = require('pg');
+
+module.exports.get = function(req, res, next) {
+  var pool = new pg.Pool(config);
+  pool.connect(function(err, client, done) {
+    if(err) {
+      return console.error('error fetching client from pool', err);
+    }
+    client.query('SELECT * FROM give_me_time_public.project', [], function(err, result) {
+      done();
+      if(err) {
+        return console.error('error running query', err);
+      }
+      var roundedRes = result.rows.map((row) => {
```

Comment:
You should extract this into a service as this "renderer" function should also be used in the GET endpoint for single projects